README.rst

```diff
@@ -99,6 +102,7 @@ And here is how you can configure different versions:
         'title'                => 'Symfony2 API',
         'build_dir'            => __DIR__.'/../build/sf2/%version%',
         'cache_dir'            => __DIR__.'/../cache/sf2/%version%',
+        'remote_repository'    => new GitHubRemoteRepository('symfony/symfony', dirname($dir)),
```

Comment:
This should probably be documented as an additional optional feature. Here, people might think this is required.